test/libsolidity/syntaxTests/using/module_4.sol

```diff
@@ -0,0 +1,18 @@
+==== Source: A ====
+function id(uint x) pure returns (uint) {
+    return x;
+}
+
+==== Source: B ====
+import "A";
+
+contract C {
+    // This only attaches the free functions in the current source unit.
+    // Therefore, `id` is not accessible.
+    using * for uint;
```

Comment:
@chriseth Is this how we want the semantics?

Reply:
I would suggest that these things are only valid at file level.

Reply:
Especially inside contracts it is horribly difficult due to inheritance.

Reply:
Disallowed this for contracts.